libs/application/templates/health-insurance/src/forms/PrerequisitesForm.ts

```diff
@@ -66,18 +69,11 @@ export const PrerequisitesForm: Form = buildForm({
             }),
             buildDataProviderItem({
               provider: UserProfileApi,
-              title: 'userProfile',
-              subTitle: 'userProfile',
-            }),
-            buildDataProviderItem({
-              id: 'healthInsurance',
-              type: 'HealthInsuranceProvider',
               title: '',
               subTitle: '',
             }),
             buildDataProviderItem({
```

Comment:
Is this one not needed anymore?

Reply:
line 75 to 79 are using HealthInsuranceApi, whic is the new refactored data provider